src/main/scala/org/constellation/p2p/Download.scala

```diff
@@ -298,7 +294,8 @@ class DownloadProcess[F[_]: Concurrent: Timer: Clock](snapshotsProcessor: Snapsh
       .liftIO(dao.snapshotService.syncBuffer.get)
       .flatMap(
         _.toList.map { h =>
-          if (!snapshotInfo.acceptedCBSinceSnapshotCache.contains(h) && !snapshotInfo.snapshotCache.contains(h)) {
+          if (!snapshotInfo.acceptedCBSinceSnapshotCache.contains(h) && !snapshotInfo.snapshotCache
+                .contains(h)) { // TODO: wkoszycki make it distinct by baseHash
```

Comment:
Can you create an issue with `technical improvement` label?

Reply:
:+1: